client/src/styles/_typography.scss

```diff
@@ -86,7 +86,7 @@ th {
   text-transform: uppercase;
   font-size: $font-size-xxs;
   font-weight: 400;
-  letter-spacing: .4px;
+  letter-spacing: .1px;
```

Comment:
`.1px` is really small, and you could end up with rounding differences across browsers with such a small value.

I believe Firefox rounds UP for this and all other browsers rounds DOWN.

Reply:
I just tested this out. It quite noticeable between `0px`, `.1px` and `1px` on Chrome and Firefox.

Reply:
I think letter spacing is rounding is fine for all browsers. Not sure why I reduced that as .4px looked fine actually (probably .3 looks even better).

Reply:
Going with .3px. Left is Chrome and right is Firefox.
![image](https://user-images.githubusercontent.com/555033/32870612-9e7bd78c-cae1-11e7-9f20-b656a0b63cbc.png)

Reply:
sweet :)